contracts/pcv/maple/MaplePCVDeposit.sol

```diff
@@ -0,0 +1,202 @@
+pragma solidity =0.8.13;
+
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {Math} from "@openzeppelin/contracts/utils/math/Math.sol";
+import {IPool} from "./IPool.sol";
+import {CoreRef} from "../../refs/CoreRef.sol";
+import {PCVDeposit} from "../PCVDeposit.sol";
+import {IMplRewards} from "./IMplRewards.sol";
+
+/// @notice PCV Deposit for Maple
+/// Allows depositing only by privileged role to prevent lockup period being extended by griefers
+
+/// Can only deposit USDC in this MAPLE PCV deposit
```

Comment:
Add comment to never connect this deposit to the allocator or any external contract that relies on the correct amount being withdrawn